bin/ch/ch.cpp

```diff
@@ -402,13 +405,23 @@ HRESULT CreateAndRunSerializedScript(const char* fileName, LPCSTR fileContents,
     IfJsErrorFailLog(ChakraRTInterface::JsGetCurrentContext(&current));
     IfJsErrorFailLog(ChakraRTInterface::JsSetCurrentContext(context));
 
+    // canonicalize that path name to lower case for the profile storage
```

Comment:
This is indeed a problem for this feature, which is why I had to capture the path before we lower it for profile purposes.